### ISSUE TITLE
<sslv3 alert handshake failure>

### ISSUE THREAD

**Issue by alfablac** [davide-romanini/comictagger#86]
It seems ComicVine changed their website today.
Can no long get ComicTagger to work. Yesterday it was fine, but not now.
I'm on Win 10, Comic tagger .15-beta 
Message on CLI is:

<urlopen error [Errno 1] _ssl.c:504: error:14077410:SSL routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake failure>

**Comment by alfablac** [comment]
Probably due to the new forced SSL implementation?

https://comicvine.gamespot.com/forums/api-developers-2334/

**Comment by afuturefoe** [comment]
Correct - since ComicTagger is using Python 2.7 and not Python 3+, the libraries used are defaulting to SSLv3 (which is blocked by ComicVine). However, I have found a way to get them to use TLS 1.0 instead.

I can provide further information and submit a pull request...

**Comment by dbb1480** [comment]
afuturefoe,

I am having the same issue.  Can you provide further information on how to use TLS 1.0

**Comment by alfablac** [comment]
If you find a way, please compile and release a new version for Windows.
All python files I have on my end are precompiled so I can't edit the code.
Tried to reinstall OpenSSL and update some libraries on Python, but couldn't get it to work.
Saw somewhere that this is a error with the urllib2 libraries. Not sure if this is going to help, but it should be just a matter of changing some calls on the API

**Comment by davide-romanini** [comment]
Maybe it's the same as #87, looking into it.

**Comment by afuturefoe** [comment]
Same as #87, yes.

I was able to get it to work with the following:

import ssl

context = ssl.SSLContext(ssl.PROTOCOL_TLSv1)
resp = urllib2.urlopen( url, context=context ) 
return resp.read()

**Comment by davide-romanini** [comment]
Thanks for the suggestion, it seems trivial to solve. Actually my problem is how to repackage the application for the various platforms, it seem I'll need lot of dependencies and the latest version of XCode refuses to install on El Capitan. This will require a little more effort..

**Comment by tommyz619** [comment]
I don't know if this will be any help, and I can't test it now, since I don't have a mac anymore, but if you need only the XCode Command Line tools, I seem to remember that I was able to get them installed from the terminal using only brew (https://brew.sh/).  Also, it may be that some of the dependencies that you needed to compile are now available  as pre-compiled packages through brew standard or third-party respositories (called Taps).

**Comment by afuturefoe** [comment]
sudo port self update
sudo port install py27-pyqt4
sudo port install py27-configparser
sudo port install py27-beautifulsoup4
sudo port install py27-pil

python2.7 comictagger.py

**Comment by davide-romanini** [comment]
Close this to move discussion on #87. I released a MacOS dmg with the fix, I hope. Let me know how it works. Many thanks @afuturefoe for your hints!
